examples/darwin-framework-tool/commands/tests/TestCommandBridge.h

```diff
@@ -101,16 +106,44 @@ class TestCommandBridge : public CHIPCommandBridge,
         SetCommandExitStatus(err);
     }
 
-    /////////// DelayCommands Interface /////////
-    void OnWaitForMs() override
+    /////////// DelayCommands /////////
+    CHIP_ERROR WaitForMs(
+        const char * _Nullable identity, const chip::app::Clusters::DelayCommands::Commands::WaitForMs::Type & value)
     {
+        [NSThread sleepForTimeInterval:value.ms / 1000];
         dispatch_async(mCallbackQueue, ^{
             NextTest();
         });
+        return CHIP_NO_ERROR;
+    }
+
+    CHIP_ERROR WaitForMessage(
+        const char * _Nullable identity, const chip::app::Clusters::DelayCommands::Commands::WaitForMessage::Type & value)
+    {
+        VerifyOrReturnError(!value.message.empty(), CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
This should have a comment about how it needs to match the DelayCommands implementation without actually compiling in DelayCommands, and a comment in DelayCommands saying it needs to match the impl here.

Reply:
Updated